### ISSUE TITLE
react-dom@17 can't be resolved

### ISSUE THREAD

**Issue by dzearing** [stackblitz/core#1905]
### **Description of Bug**
React 17 scenarios are totally blocked on stackblitz.

### **Steps to Reproduce**

1. Create a new React project
2. Dependencies default to React 18.
3. Add `react-dom@17`

### **Expected Behavior**
react-dom@17.0.2 can be installed

### **Screenshots/Screencast**

![image](https://user-images.githubusercontent.com/1110944/174870549-ae490906-d288-41e3-a7c4-b3cdcac54e7b.png)

**Comment by dzearing** [comment]
Here's an example project. https://stackblitz.com/edit/react-ts-1jwzyx

**Comment by dzearing** [comment]
@fvsch can you reopen or should i open a new issue? :)

**Comment by dzearing** [comment]
Nevermind. I see what you mean! The error message is misleading - it's installing it, it can resolve `react-dom`, but it can't resolve the specific import `react-dom/client` in the `index.tsx`.

**Comment by Hotell** [comment]
hey folks. wondering why was this closed ? its still doesn't work 

<img width="2262" alt="image" src="https://github.com/stackblitz/core/assets/1223799/7de447a7-184e-47dc-8f06-4cbcb53fd697">

https://stackblitz.com/edit/react-uratzx?file=App.tsx

**Comment by CrypticArtsLaboratory** [comment]
example is 404.. https://stackblitz.com/edit/react-17

been tracking since Jun 29, 2022

![image](https://github.com/stackblitz/core/assets/5160950/e2ab284c-6f60-4b81-a961-ef0bdd6d9769)
